workspaces/ui-v2/src/optic-components/documentation/ContributionGroup.tsx

```diff
@@ -58,7 +59,9 @@ function createFlatList(
       });
     }
     if (shape.asArray) {
-      contributions.push(...createFlatList(shape.asArray.shapeChoices, depth + 1));
+      contributions.push(
```

Comment:
Prettier formatting here - I'm going to look into adding prettier on commit (on staged files) since it's easy to either not have prettier set up to format on save, or to forget to prettier something